src/EA.Weee.Web/Areas/Admin/Views/ManageEvidenceNotes/ViewAllEvidenceNotes.cshtml

```diff
@@ -56,22 +56,22 @@
                             <caption class="govuk-table__caption govuk-visually-hidden">All evidence notes</caption>
                             <thead class="govuk-table__head">
                                 <tr class="govuk-table__row">
-                                    <th class="govuk-table__header" scope="col">
+                                    <th class="govuk-table__header" scope="col" style="vertical-align: bottom; width: 10%;">
                                         Reference ID
                                     </th>
-                                    <th class="govuk-table__header" scope="col">
+                                    <th class="govuk-table__header" scope="col" style="width: 15%;">
                                         Submitted date
                                     </th>
-                                    <th class="govuk-table__header" style="vertical-align: bottom;" scope="col">
+                                    <th class="govuk-table__header" scope="col" style="vertical-align: bottom; width: 30%;">
```

Comment:
@GuidoWorkDev  Ill fix this but not sure why the vertical align styles were added to the columns...also the % widths didnt add up to 100% they came to 105%...